src/random.c

```diff
@@ -11,7 +11,12 @@
 
 #include <stdint.h>
 
-int fido_get_random(void *buf, size_t len) {
+#if defined(NO_SOFTWARE_RNG)
+fido_get_random_t fido_get_random = NULL;
```

Comment:
Why include this if is overwritten anyway?

Reply:
Actually, you need this location to be defined (in a C file) in order to have the symbol within the library. Otherwise, you wouldn't be able to access the function pointer via the `extern ...`.